base/services/tftpd/tftpd.cpp

```diff
@@ -543,7 +543,7 @@ void processRequest(void *lpParam)
 #ifdef __REACTOS__
                 for (MYWORD j = 0; j < _countof(cfig.hostRanges) && cfig.hostRanges[j].rangeStart; j++)
 #else
-                for (int j = 0; j <= 32 && cfig.hostRanges[j].rangeStart; j++)
+                for (int j = 0; j < 32 && cfig.hostRanges[j].rangeStart; j++)
```

Comment:
Did you miss to read line 543?

Reply:
Oh, this is thirdparty code?

Reply:
Probably a very old one, but yeah, it's treated like that.
So the part inside `#ifdef __REACTOS__` compiles, which is fine